modules/aws_account_permissions/variables.tf

```
@@ -0,0 +1,32 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Required PARAMETERS
+# These variables have default values and can be modified based on the requirement.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+variable "iam_role_name" {
+  description = "Customize the name of IAM role for Illumio AWS integration"
+  type        = string
+  default     = "IllumioCloudIntegrationRole"
```

None of the names of resources must be hardcoded. That includes every `aws_iam_role`, every `aws_iam_role_policy`s, etc.
Instead of having one variable for each resource name, redesign this variable into a name prefix that will be used to name every AWS resource.
```suggestion
variable "iam_name_prefix" {
  description = "The prefix given to all AWS IAM resource names."
  type        = string
  default     = "IllumioCloudIntegration"
```

---

I fixed this.